src/components/Table/GridTable.test.tsx

```diff
@@ -581,6 +582,40 @@ describe("GridTable", () => {
     });
   });
 
+  describe("gtc", () => {
+    it("as=div defaults to auto", () => {
+      expect(calcGridColumns("div", [{}, {}], undefined)).toEqual("auto auto");
+    });
+
+    it("as=virtual defaults to percentage widths", () => {
+      expect(calcGridColumns("virtual", [{}, {}], undefined)).toEqual("50% 50%");
+    });
+
+    it("as=div treats numbers as fr", () => {
+      expect(calcGridColumns("div", [{ w: 1 }, { w: 2 }] as any, undefined)).toEqual("1fr 2fr");
+    });
+
+    it("as=virtual treats numbers as pixels", () => {
+      expect(calcGridColumns("virtual", [{ w: 1 }, { w: 2 }] as any, undefined)).toEqual("1px 2px");
+    });
+
+    it("as=virtual accepts percentages ", () => {
+      expect(calcGridColumns("virtual", [{ w: "10%" }, { w: 2 }] as any, undefined)).toEqual("10% 2px");
+    });
+
+    it("as=virtual rejects relative units", () => {
+      expect(() => calcGridColumns("virtual", [{ w: "10fr" }, { w: 2 }] as any, undefined)).toThrow(
+        "as=virtual only supports px or percentage units",
+      );
+    });
+
+    it("as=virtual with both px and default", () => {
+      expect(calcGridColumns("virtual", [{ w: 200 }, {}, {}] as any, undefined)).toEqual(
+        "200px 33.33333333333333% 33.33333333333333%",
```

Comment:
@bdow @KoltonG how fancy do you think we should get in this scenario? I.e.

* Leave as-is, it makes sense?
* Change it to `200px 50% 50%`
* Change it to `200px calc((100% - 200px) / 2) calc((100% - 200px) / 2)` (I don't even know if that is possible)

Reply:
Oh hey, the 3rd option is doable, so added that...

Reply:
...I think I just re-implemented the `fr` unit...